.github/actions/devcontainer_run_command/action.yml

```diff
@@ -174,5 +177,6 @@ runs:
             && inputs.CORE_APP_SERVICE_PLAN_SKU) || 'P1v2' }}" \
           -e WORKSPACE_APP_SERVICE_PLAN_SKU="${{ (inputs.WORKSPACE_APP_SERVICE_PLAN_SKU != ''
             && inputs.WORKSPACE_APP_SERVICE_PLAN_SKU) || 'P1v2' }}" \
+          -e TF_VAR_resource_processor_number_processes_per_instance="${{ inputs.RESOURCE_PROCESSOR_NUMBER_PROCESSES_PER_INSTANCE }} \
```

Comment:
You probably need to specifiy a default here as well, otherwise you must always create the github secret (which isn't something we want...)